app/src/main/java/org/oppia/android/app/profile/ProfileInputView.kt

```diff
@@ -60,9 +60,9 @@ class ProfileInputView @JvmOverloads constructor(
     orientation = VERTICAL
     if (
       attributes.getBoolean(
-        R.styleable.ProfileInputView_isPasswordInput,
-        /** defVal= */ false
-      )
+          R.styleable.ProfileInputView_isPasswordInput,
```

Comment:
@soamOne You have changed indentation here, fixing that would solve the ktlint error